.docksal/commands/release

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+## Make a new release and deploys it.
+##
+## Usage: fin release
+
+# Abort if anything fails
+set -e
+
+# Backup the environments.
+fin terminus backup:create ${hostingplatform}.dev --keep-for=365
+fin terminus backup:create ${hostingplatform}.test --keep-for=365
+fin terminus backup:create ${hostingplatform}.live --keep-for=365
+
+# Grab the live db.
+fin pull db -y --hosting-env=live
+
+# @TODO Need an IF statement here if there is any changes.
+# This is an idea:
+# var_config_status=$(fin drush config:status 2>&1 | grep "No differences")
+# if [[ "$var_config_status" != "[notice] No differences between DB and sync directory." ]]
+# then
+#     echo -e "If statement."
+# else
+#     echo -e "Else statement"
+# fi
+# echo -e "Statement: $var_config_status"
+# =================================================================
+
+# Make a new branch.
+git checkout -b feature/live-config
+
+# Export and commit config.
+fin drush config:export -y --commit --message="Automated configuration capture from live database"
```

Comment:
We talked about this but think bringing this into it's own command could be better all around and not leave for possible issues.